applications/dashboard/scss/editor/embeds/_embedGlobalVariables.scss

```diff
@@ -1,4 +1,4 @@
 $embedded-error_bg: rgba(255,57,52,0.12) !default;
```

Comment:
Should be derived from `$global-error_fg` (mix `$global-color_fg` and/or `$global-color_fg`.

Reply:
Done